Jenkinsfile

```diff
@@ -102,6 +102,7 @@ pipeline {
                         sh 'cd build && make check-clang-tidy'
                         sh 'cd build && gtimeout 1h make unittest'
                         sh 'cd build && gtimeout 1h make check-tpl'
+                        // sh 'cd build && gtimeout 10s python3 ../script/testing/kill_server_on_port.py 15721'
```

Comment:
Add a comment to explain why this commented out.

Reply:
commented code removed